ios/engine/KMEI/KeymanEngine/Classes/SettingsViewController.swift

```diff
@@ -469,3 +478,9 @@ open class SettingsViewController: UITableViewController {
     }
   }
 }
+
+extension SettingsViewController: UIAdaptivePresentationControllerDelegate {
+  public func presentationControllerWillDismiss(_ presentationController: UIPresentationController) {
+    doneClicked(1)
```

Comment:
I feel like we should rename that method because we are now calling it when done was not actually clicked. The comment for the method says it resets KMW, so maybe applySettings or something similar.

What does it mean to pass 1 to it?

Reply:
The method was written to take a single value of 'any' type that was subsequently ignored.  Sadly, `nil` wasn't accepted; I tried that first.  I can try investigating why the method takes an argument and see about eliminating it there.

Fair point on renaming the method.  Pretty sure this area was originally your addition; any preferences on nomenclature?  I could see calling it `finalize` or `onDismiss`, to put a couple of ideas out there.

Reply:
How about `settingsDismissed`?